Cargo.toml

```diff
@@ -176,7 +176,7 @@ once_cell = { version = "1.5.2", default-features = false, features=["std"] }
 web-sys = { version = "0.3.37", default-features = false, features = ["Crypto", "Window"] }
 
 [target.'cfg(target_os = "windows")'.dependencies]
-winapi = { version = "0.3.8", default-features = false, features = ["ntsecapi", "wtypesbase"] }
```

Comment:
Shouldn't we add this feature only for aarch64 targets? I think this would require adding `resolver = "2"` to Cargo.toml too?

Reply:
Adding a single winapi module for all Windows targets is no big deal, what may happen is we'd forget to re-enable it if we use it on Windows x64. Is it even possible to tweak the Cargo.toml to limit just one such feature to Windows ARM64? As for the resolver=2, I do not know what you are talking about

Reply:
> Adding a single winapi module for all Windows targets is no big deal

I agree.

> Is it even possible to tweak the Cargo.toml to limit just one such feature to Windows ARM64? As for the resolver=2, I do not know what you are talking about

Yes, starting in Rust 1.151: https://doc.rust-lang.org/cargo/reference/features.html#feature-resolver-version-2

Reply:
I am fine to leave this the way you did it